docs/concept-overview.md

```diff
@@ -35,6 +35,14 @@ It preserves the full application state, including function stacks with local va
 
 A [Temporal SDK](/docs/sdks-introduction) then enables you to write your application code using the full power of the programming language, while the Temporal Server handles the durability, availability, and scalability of the application.
 
+:::note Temporal as a Distributed System
+In terms of CAP theorem, each Temporal cluster is eventually available and highly consistent.
+
+- Because Temporal makes it easy to retry activities and horizontally scale resources, availability loss doesn't result in a fault, but in increased latency.
```

Comment:
```suggestion
- Because Temporal makes it easy to retry Activities and horizontally scale resources, availability loss doesn't result in a fault, but in increased latency.
```

Capitalize core concepts as proper nouns.